picatrix/lib/manager.py

```diff
@@ -111,6 +171,28 @@ def get_magic_info(cls, as_pandas: bool = True) -> Union[
     df = pandas.DataFrame(entries)
     return df[cls.MAGICS_DF_COLUMNS].sort_values('name')
 
+  @classmethod
+  def register_helper(
+      cls, name: Text, helper: Any, typing_help: Dict[Text, Any]):
+    """Register a picatrix helper function.
+
+    Args:
+      name (str): the name of the helper function.
+      helper (function): the helper function to register.
+      typing_help (dict): dict with the arguments and their types.
+
+    Raises:
+      KeyError: if the helper is already registered.
+    """
+    if name in cls._helpers:
+      raise KeyError(
+          f'The helper [{name}] is already registered.')
+    cls._helpers[name] = {
```

Comment:
I'd suggest you turn define a Helper record/struct as mentioned above.

Reply:
ack